traffic_ops/testing/api/about_test.go

```diff
@@ -0,0 +1,33 @@
+package v13
```

Comment:
Can you also make these `package api`. The package name `api` isn't great (like `v13`), we should probably make this something like `totestapi`; but we can decide that later. For now, it just needs to match the directory, so the package import and usage isn't confusing.

Reply:
Removing the testing for now so that it can be discussed later. I'm keeping it v13 because that is what it was originally.